build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id "eu.xenit.docker-compose" version "5.2.0" apply false
+    id "eu.xenit.docker-alfresco" version "5.2.0" apply false
```

Comment:
`eu.xenit.docker-compose` & `eu.xenit.docker-alfresco` plugins are packaged in the same jar, so you don't need to apply them both separately.

Reply:
aha, cool

Do we have a preference for which one we usually apply?
Or do I just pick one?

Reply:
If you use `apply false`, it just adds the jar on the classpath, so from a technical standpoint it does not matter which one we apply (well, actually which one we *don't* apply, but which one we add on the plugin classpath).
If you're only using one of the plugins in the whole project, I would take that one.
If you're using multiple of the plugins in the project, I pick the first one that I start using or find. There is not really any preference, and you will get a clear error message from Gradle when you're trying to mix different versions of the same jar.

Reply:
Just to be clear, this only applies here because of the `apply false`?

Because according to this page, you still need to apply them both if you need `composeUp` or `composeDown` tasks
https://github.com/xenit-eu/alfresco-docker-gradle-plugin/blob/master/UPGRADING-5.0.md

Reply:
1. Yes this only applies here because of the `apply false`.
2. You linked a migration guide, not the primary documentation.
    Only the `eu.xenit.docker-compose` creates `composeUp`/`composeDown` tasks. In 4.x, these tasks were created by `eu.xenit.docker` & `eu.xenit.docker-alfresco` and there was no way to create a project with just docker-compose tasks. In 5.0, we chopped the monolithic plugin into pieces that can be used individually. `eu.xenit.docker` & `eu.xenit.docker-alfresco` no longer create docker-compose tasks, so to keep the same functionality, you would then have to apply the `eu.xenit.docker-compose` plugin.

Reply:
Thanks for the explanation!